DOCSUP-12003: S3 and HDFS zero copy replication (#27055)
ClickHouse/ClickHouse: docs/en/engines/table-engines/integrations/hdfs.md

## Implementation Details {#implementation-details}

-   Reads and writes can be parallel.

> -   Zero-copy replication is supported, which means that if the data is stored remotely on several machines and needs to be synchronized, then only the metadata is replicated (paths to the data parts), but not the data itself.  

Reviewer: This file is not a proper place for this info because it is very remotely related to table engines. A better place would be where DiskHDFS is documented or in some replication.md or similar place.

"machines" is quite informal. Probably better to omit this part because it is obvious from term of replication:
```suggestion
-   Zero-copy replication is supported, which means that only the metadata is replicated (paths to the data parts located in HDFS), but not the data itself.
```

Author: @kssenii There is no special place in the ClickHouse docs where s3 or HDFS disks are documented, only in relation with the table engines (and that is quite okay, to my mind). So I'm afraid this is the best place for now.

Reviewer: should this https://github.com/ClickHouse/ClickHouse/pull/26324/files#diff-4c0afa21296979beb16ebfb9d9cc693f769762d1a7a5a6afc6c08ab40ca1113fR813 also be moved there?

Author: @kssenii Yes, this would be done in the other PR that is coming (I mentioned it earlier).
